azurelinuxagent/common/event.py

```diff
@@ -77,6 +77,8 @@ class WALAEventOperation:
     CGroupsInfo = "CGroupsInfo"
     CGroupsInitialize = "CGroupsInitialize"
     CGroupsLimitsCrossed = "CGroupsLimitsCrossed"
+    CollectEventErrors = "CollectEventErrors"
```

Comment:
@pgombar gave me the idea of reporting the number of dropped events per run to Kusto so that we have some data on how many events our agent is dropping. To achieve this I broke down the errors into these 4 categories. The events can either be dropped either while collecting them from filesystem or while sending them to Wireserver. The sum of all 4 of these can specify the total no of dropped events. 

I broke it down into 4 different parts mainly to have more granularity over the errors that we see and hence can work towards understanding the problem better.